### PR TITLE
Add SEGMENTATION_TYPE to enum

### PR DESCRIPTION
**Pull request by LucasMMota**
**What**
Adding new method to TailoredAudience: all

Also adding `SEGMENTATION_TYPE` enum (according to docs [metrics-and-segmentation](https://developer.twitter.com/en/docs/ads/analytics/overview/metrics-and-segmentation#segmentation)).

### CODE REVIEW

File: twitter_ads/enum.py [hunk 1]
```diff
@@ -195,3 +195,25 @@ def enum(**enums):
...
+    INTERESTS='INTERESTS',
+    KEYWORDS='KEYWORDS',
+    LANGUAGES='LANGUAGES',
+    LOCATIONS='LOCATIONS',
+    PLATFORMS='PLATFORMS',
+    PLATFORM_VERSIONS='PLATFORM_VERSIONS',
+    POSTAL_CODES='POSTAL_CODES',
+    REGIONS='REGIONS',
+    SIMILAR_TO_FOLLOWERS_OF_USER='SIMILAR_TO_FOLLOWERS_OF_USER',
+    TV_SHOWS='TV_SHOWS',
```
Comment (smaeda-ks) [inline twitter_ads/enum.py]: Can you remove the trailing comma here? `,`

File: twitter_ads/audience.py [hunk 2]
```diff
@@ -22,6 +22,17 @@ class TailoredAudience(Resource):
     RESOURCE_PERMISSIONS = '/' + API_VERSION + '/accounts/{account_id}/tailored_audiences/\
 {id}/permissions'
 
+    @classmethod
+    def all(klass, account, **kwargs):
```
[the remark this answers is not in the document]
Comment (LucasMMota) [reply twitter_ads/audience.py]: Sure!

File: twitter_ads/enum.py (hunk 1 above)
Comment (LucasMMota) [reply twitter_ads/enum.py]: :+1:  removed

**Comment by smaeda-ks** [comment]
@LucasMMota Thanks for the update! LGTM

**Pull request merged**